api/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+from . import views
+
+urlpatterns = [
+    path("v1/hunt/<int:pk>", views.HuntAPIView.as_view(), name="hunt_api_view"),
```

Comment:
Open to suggestions-- I'm not sure what that might look like since I don't think we'd want to use a CURRENT_API_VERSION-type constant since the reason to version the API to begin with is to separate these v1 endpoints from any future v2, and some V1_API="v1" type thing doesn't feel like an improvement

Reply:
Oh yeah, using a constant won't make as much sense if we want to maintain the various version endpoints. 

Maybe we can use `include()` to nest URLs? (see end of this section: https://docs.djangoproject.com/en/3.1/topics/http/urls/#including-other-urlconfs) That might at least deduplicate the "v1" string for each path by grouping them all together.

tbh, it's probably fine to leave these routes as is, and address organization of the version endpoints once we actually switch versions 😄